src/platform/packages/shared/presentation/presentation_containers/interfaces/unsaved_changes/children_unsaved_changes.test.ts

```diff
@@ -13,20 +13,22 @@ import { waitFor } from '@testing-library/react';
 
 describe('childrenUnsavedChanges$', () => {
   const child1Api = {
-    unsavedChanges$: new BehaviorSubject<object | undefined>(undefined),
+    uuid: 'child1',
+    hasUnsavedChanges$: new BehaviorSubject<boolean>(false),
     resetUnsavedChanges: () => true,
```

Comment:
resetUnsavedChanges no longer needs to return anything, so change to resetUnsavedChanges: () => undefined